common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -478,7 +478,7 @@ def multi_directory_group_intro
                        "`#{dep.humanized_version}`"
                      ]
                    end
-                   "\n\n#{table([header] + rows)}"
+                   "\n\n#{table([header] + rows)}\n"
```

Comment:
I realized that my refactored code was adding extra newlines and few of the smoke tests where failing so in-order to fix it,  I added `\n` at the end of the `"\n\n#{table([header] + rows)}"` statement in my refactored code. The statement was also present in the old code as well so I just reverted all my changes and simply added `\n` at the end of the of `"\n\n#{table([header] + rows)}\n"` and it fixes all the issues

Reply:
Tested locally and it correctly creates the table and separates directory dependencies correctly 

```bash
                Bumps the npm_and_yarn group with 5 updates in the /npm/multi-dir/grouped-security-update/backend directory:

                | Package | From | To |
                | --- | --- | --- |
                | [fast-jwt](https://github.com/nearform/fast-jwt) | `3.0.0` | `3.3.2` |
                | [follow-redirects](https://github.com/follow-redirects/follow-redirects) | `1.13.3` | `1.15.6` |
                | [payload](https://github.com/payloadcms/payload) | `1.5.9` | `1.7.0` |
                | [yaml](https://github.com/eemeli/yaml) | `2.1.0` | `2.2.2` |
                | [mongodb](https://github.com/mongodb/node-mongodb-native) | `4.8.1` | `` |
                | [payload](https://github.com/payloadcms/payload) | `1.7.0` | `2.11.2` |

                Bumps the npm_and_yarn group with 4 updates in the /npm/multi-dir/grouped-security-update/frontend directory: [follow-redirects](https://github.com/follow-redirects/follow-redirects), [braces](https://github.com/micromatch/braces), [next](https://github.com/vercel/next.js) and [jquery](https://github.com/jquery/jquery).

                Updates `fast-jwt` from 3.0.0 to 3.3.2
```

Reply:
@Nishnha @jakecoffman is it ok to ship it without spec it is now just adding a newline character?

Reply:
Nevermind, I have added test as well